api/pom.xml

```diff
@@ -93,48 +93,58 @@
     </developers>
 
     <properties>
+        <release.spec.feedback>jsonb-dev@eclipse.org</release.spec.feedback>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <last.final.spec.version>0.9</last.final.spec.version>
         <milestone.number>01</milestone.number>
-        <next.final.spec.version>1.0</next.final.spec.version>
-        <new.spec.version>${next.final.spec.version}</new.spec.version>
-        <non.final>true</non.final>
+        <new.spec.version>1.1.0</new.spec.version>
+        <non.final>false</non.final>
         <skip.release.tests>false</skip.release.tests>
-        <spec.version>${last.final.spec.version}</spec.version>
-        <javax.json.version>1.1</javax.json.version>
+        <spec.version>1.1</spec.version>
+        <jakarta.json.version>1.1.5</jakarta.json.version>
         <api_package>javax.json.bind</api_package>
+        <legal.doc.source>${project.basedir}/..</legal.doc.source>
+        <vendor.name>Oracle Corporation</vendor.name>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>jakarta.json</groupId>
+                <artifactId>jakarta.json-api</artifactId>
+                <version>${jakarta.json.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.12</version>
+                <scope>test</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
     <dependencies>
         <dependency>
-            <groupId>javax.json</groupId>
-            <artifactId>javax.json-api</artifactId>
-            <version>${javax.json.version}</version>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
         </dependency>
     </dependencies>
 
     <profiles>
         <profile>
             <id>jdk9</id>
             <activation>
-                <jdk>[1.9,)</jdk>
+                <jdk>[9,)</jdk>
             </activation>
             <build>
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
why are you keeping empty definition here?

Reply:
It was not me who added it. :)
It's gone.